libraries/botframework-connector/src/auth/jwtTokenExtractor.ts

```diff
@@ -173,23 +187,17 @@ export class JwtTokenExtractor {
             }
 
             if (this.tokenValidationParameters.algorithms) {
-                if (this.tokenValidationParameters.algorithms.indexOf(decodedToken.header.alg) === -1) {
+                if (this.tokenValidationParameters.algorithms.indexOf(header.alg) === -1) {
                     throw new AuthenticationError(
-                        `"Token signing algorithm '${decodedToken.header.alg}' not in allowed list`,
+                        `"Token signing algorithm '${header.alg}' not in allowed list`,
                         StatusCodes.UNAUTHORIZED
                     );
                 }
             }
 
-            const claims: Claim[] = Object.keys(decodedPayload).reduce((acc: any, key: any) => {
-                acc.push({ type: key, value: decodedPayload[key] });
-
-                return acc;
-            }, <Claim[]>[]);
-
-            return new ClaimsIdentity(claims, true);
+            const claims = Object.entries(decodedPayload).map<Claim>(([type, value]) => ({ type, value }));
+            return new ClaimsIdentity(claims, true); // TODO(joshgummersall): fix true
```

Comment:
@stevengum unclear what the auth type should be here. Any thoughts?

Reply:
@gabog @EricDahlvang do either of you have any input on this? TBH I'm not sure what the [AuthenticationType](https://docs.microsoft.com/en-us/dotnet/api/system.security.principal.windowsidentity.authenticationtype?view=netcore-3.1#System_Security_Principal_WindowsIdentity_AuthenticationType) normally is for activities from the Channel in .NET, or if the value changes between environments/callers.

Reply:
As discussed with @gabog we will leave this single instance of `true` as it doesn't quite make sense to port over the various authentication constants.